Frontend/RestaurantDashboard/src/pages/MenuList.tsx

```diff
@@ -2,15 +2,18 @@ import { useEffect, useState } from "react";
 import { Pencil, Trash2 } from "lucide-react";
 import { Switch } from "@headlessui/react";
 import React from "react";
+import { useNavigate } from 'react-router-dom';
 
 type MenuItem = {
+  id: number;
```

Comment:
O campo `id` deve ser `string`

```suggestion
  id: string;
```